src/pages/iou/request/step/IOURequestStepWaypoint.tsx

```diff
@@ -195,45 +202,46 @@ function IOURequestStepWaypoint({
                     danger
                 />
                 <FormProvider
-                    style={[styles.flexGrow1, styles.mh5]}
+                    style={[styles.flex1, styles.mh5]}
                     formID={ONYXKEYS.FORMS.WAYPOINT_FORM}
                     enabledWhenOffline
                     validate={validate}
                     onSubmit={submit}
+                    submitFlexEnabled={false}
+                    // This keeps the list of waypoints above the keyboard on android
+                    submitButtonStyles={keyboardHeight && Platform.OS === 'android' ? {height: keyboardHeight} : null}
```

Comment:
This calculation doesn't seem to be correct, there is a gap between the list and a keyboard (Pixel 6 simulator, Android 13)

https://github.com/Expensify/App/assets/9059945/59e0435a-d2e4-41f1-875f-bb2b5f620ca8